src/app/(sidebar)/account/muxed-create/page.tsx

```diff
@@ -1,5 +1,213 @@
+              value={baseAddress}
+              onChange={(e) => {
+                setReset(true);
+                setBaseAddress(e.target.value);
+
+                let error = "";
+
+                if (e.target.value.startsWith("M")) {
+                  error = "Base account address should start with G";
+                } else {
+                  error = validate.publicKey(e.target.value) || "";
+                }
+
+                setBaseFieldErrorMessage(error);
+              }}
+              error={baseFieldErrorMessage}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <Input
+              id="muxed-account-iD"
+              fieldSize="md"
+              placeholder="Ex: 1"
+              label="Muxed Account ID"
+              value={muxedId}
+              onChange={(e) => {
+                setReset(true);
+                setMuxedId(e.target.value);
+
+                const error = validate.positiveInt(e.target.value);
+                setMuxedFieldError(error || "");
+              }}
+              error={muxedFieldError}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <div className="Account__CTA">
+              <Button
+                disabled={
+                  !baseAddress ||
+                  !muxedId ||
+                  Boolean(baseFieldErrorMessage || muxedFieldError)
+                }
+                size="md"
+                variant={"secondary"}
+                onClick={generateMuxedAccount}
+              >
+                Create
+              </Button>
+            </div>
+          </div>
+        </div>
+
+        <ExpandBox
+          isExpanded={
+            !isReset && Boolean(account.generatedMuxedAccount.muxedAddress)
+          }
+        >
+          <div className="Account__content__inputs Account__content__inputs--grey">
+            <PubKeyPicker
+              id="muxed-public-key-result"
+              label="Base Account G Address"
+              value={account.generatedMuxedAccount.baseAddress || ""}
+              error=""
+              readOnly={true}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <Input
+              id="muxed-account-id-result"
+              fieldSize="md"
+              label="Muxed Account ID"
+              value={account.generatedMuxedAccount.id}
+              error=""
+              readOnly={true}
+              copyButton={{
+                position: "right",
+              }}
+            />
+
+            <PubKeyPicker
+              id="muxed-account-address-result"
+              label="Muxed Account M Address"
+              value={account.generatedMuxedAccount.muxedAddress || ""}
+              error=""
+              readOnly={true}
+              copyButton={{
+                position: "right",
+              }}
+            />
+          </div>
```

Comment:
How about we move this to its own component with props for G-address, M-address, and ID? It's a static component we can use to create and process muxed accounts. The forms are identical on both screens.

<details>
<summary>Screenshots</summary>

![image](https://github.com/stellar/laboratory/assets/7346473/9c98bbf3-bfdd-4fd8-998d-ccf92190028c)

![image](https://github.com/stellar/laboratory/assets/7346473/94852010-5f25-47ec-ad9d-b32188f6b2d6)
</details>